test/features/formItemExtra.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS } from '../factories';
+import { IValue } from '../../src/props';
+
+const SUPPORTING_COMPONENTS = [
+  'Form',
+  'FormCard',
+  'FormDrawer',
+  'FormModal',
+];
+
+function testFormat (value: IValue) {
```

Comment:
This formatter can be named `extraRenderer` or something along those lines for clarity's sake (it could also be defined inline)